.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,106 @@
+  - type: dropdown
+    id: tool
+    attributes:
+      label: Select the SuiteCloud tool where you encountered the issue
+      options:
+        - SuiteCloud CLI
+        - SuiteCloud Extension for VSCode
+        - SuiteCloud Unit Testing
+    validations:
+      required: true
+  - type: input
+    id: toolVersion
+    attributes:
+      label: Version of the SuiteCloud Tool
+      description: |
+                    For CLI run: `suitecloud --version`
+
+                    For VSCode Extension: `ctrl+shift+x` , search for suitecloud extension, Check the Version in front of Extension Name (Not in the description below)
+
+                    For Unit-Testing package: Open package.json from your suitecloud project, check the version on "@oracle/suitecloud-unit-testing": ^1.2.1"
+      placeholder: "1.2.0"
+    validations:
+      required: true
+  - type: input
+    id: os
+    attributes:
+      label: Operating system
+      description: "Which operating system do you use? Please provide the version as well."
+      placeholder: "macOS Big Sur 11.5.2"
+    validations:
+      required: true  
+  - type: input
+    id: Node
+    attributes:
+      label: Node Version/ VSCode Version
+      description: |
+        Please provide the full Node version of your project. node -v 
+           OR
+        Please provide VSCODE version in case you selected SuiteCloud Extension. This can be checked in "About VIsual Studio Code".
+      placeholder: "v18.15.0, 1.78.0 (Universal)"
+    validations:
+      required: true
+  - type: input
+    id: java
+    attributes:
+      label: JAVA Version
+      description: "Please provide the full Java version. java --full-version"
+      placeholder: "java version '17.0.6 2023-01-17 LTS'"
+    validations:
+      required: true
+  - type: textarea
+    id: bug-description
+    attributes:
+      label: Bug description
+      description: What happened?
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Steps To Reproduce
+      description: Steps to reproduce the behavior.
+      placeholder: |
+        1. when running suitecloud account:setup ...
+        2. creating a new authid
+        3. Browser based authentication
+        4. See error...
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Actual Output
+      description: |
+        Please provide the actual result/output
+
+        Tip: You can attach images or log files by clicking this area to highlight it and then dragging files in.
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Expected Output
+      description: |
+        Please provide the expected result/output
+
+        Tip: You can attach images or log files by clicking this area to highlight it and then dragging files in.
+    validations:
+      required: false 
+  - type: textarea
+    attributes:
+      label: Anything else?
```

Comment:
I'm fine as it is now, but what do you think about renaming this to `Extra relevant information`